src/main/kotlin/calculator/parser/StringSplitModule.kt

```diff
@@ -0,0 +1,32 @@
+package calculator.parser
+
+class StringSplitModule {
+
+    private val regex = Regex("//(.)\n(.*)")
+
+    fun split(text: String): List<Int> {
+        val result = regex.find(text)
+        result?. let {
+            val customDelimiter = it.groupValues[1]
+            val numberList = it.groupValues[2].split(customDelimiter)
+            return numberList.map { str: String -> str.toPositiveNumber() }
+        }
+        return defaultDelimiter(text)
+    }
+
+    private fun defaultDelimiter(text: String): List<Int> {
+        return text.split(*DEFAULT_DELIMITER).map { it.toPositiveNumber() }
+    }
+
+    private fun String.toPositiveNumber(): Int {
+        val number = this.trim().toInt()
+        if (number < MIN_NUM) throw RuntimeException(OUT_OF_RANGE)
+        return number
+    }
+
+    companion object {
+        val DEFAULT_DELIMITER = arrayOf(",", ":")
+        const val MIN_NUM: Int = 0
+        const val OUT_OF_RANGE: String = "음수를 입력하실수 없습니다."
```

Comment:
```suggestion
        private val DEFAULT_DELIMITER = arrayOf(",", ":")
        private const val MIN_NUM: Int = 0
        private const val OUT_OF_RANGE: String = "음수를 입력하실수 없습니다."
```
해당 클래스 내부에서만 쓰이는 것이라면 반드시 private 접근자를 붙여주세요